runtime/router.go

```diff
@@ -134,77 +55,55 @@ func NewRouterEndpoint(
 	handlerName string,
 	handler HandlerFn,
 ) *RouterEndpoint {
-	endpointTags := map[string]string{
+	logger := gateway.Logger.With(
+		zap.String("endpoint", endpointName),
```

Comment:
can we make this `endpointID` and `handlerID`

Reply:
Done.